src/styles/utils/_variables.scss

```
@@ -0,0 +1 @@
+$transition-card-duration: 300ms;
```

You can shorten it, but it's up to you
```suggestion
$transition-duration: 300ms;
```